src/test/addresses_wallets.sol

```diff
@@ -122,6 +134,9 @@ contract Wallets {
         // Ecosystem Scope
         addr["ECOSYSTEM_SCOPE_WALLET"]       = 0x6E51E0b5813152880C1389E3e860e69E06aD04D9;
 
+        // Accessibility Scope
+        addr["LAUNCH_PROJECT_FUNDING"]       = 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F;
```

Comment:
Noting for a future PR or spell that this might be chosen to be renamed `LAUNCH_PROJECT_WALLET` to match the ecosystem scope wallet above.

Not a blocking change and I am not requesting it this spell, this is just for future spell teams.